src/Build/System.Text.Encodings.Web.pkgdef

```diff
@@ -1,7 +0,0 @@
-[$RootKey$\RuntimeConfiguration\dependentAssembly\bindingRedirection\{1A1A9DA4-9F25-4AC8-89BF-BCEF74875CA8}]
```

Comment:
Can you also delete the references to these in our .swr too?

https://github.com/dotnet/msbuild/blob/2a7dadfc62c8bdb04d4f7b9234006920a7a85de5/src/Package/MSBuild.VSSetup/files.swr#L313-L314

Reply:
Was just looking for those!